.github/workflows/node.js.yml

```diff
@@ -5,6 +5,8 @@ on:
     branches: [ main ]
   pull_request:
     branches: [ main ]
+  schedule:
+    - cron:  '30 17 * * *'
```

Comment:
Note [this is in UTC](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#scheduled-events) so would run at 10:30am PDT.

I don't mind when it runs, just wanted to make sure that's intended.

Reply:
Yeah, I don't know what time would be best. I didn't want us to ignore it. But maybe running after the workday (6pm) would be better

Reply:
Idk. ~10:30am is at the same time as stand-ups, so we would see it when filling out stand-up statuses.